src/app/groups/containers/managed-group-list/managed-group-list.component.ts

```diff
@@ -15,19 +27,30 @@ import { GroupManagershipLevel, groupManagershipLevelEnum as l } from '../../mod
   imports: [
     NgIf,
     ErrorComponent,
-    TableModule,
-    SharedModule,
     RouterLink,
     NgClass,
+    CdkTable,
+    CdkRow,
+    CdkRowDef,
+    CdkNoDataRow,
+    CdkColumnDef,
+    CdkCell,
+    CdkCellDef,
+    CdkHeaderCell,
+    CdkHeaderCellDef,
+    CdkHeaderRow,
+    CdkHeaderRowDef,
+    LoadingComponent,
   ],
 })
 export class ManagedGroupListComponent implements OnInit {
 
   state: 'error' | 'ready' | 'fetching' = 'fetching';
-  currentSort: string[] = [];
 
   data: Group[] = [];
 
+  displayedColumns = signal([ 'name', 'type', 'canManage', 'canWatchMembers', 'canGrantGroupAccess' ]);
```

Comment:
I don't understand the use of that as it is not modified...  but whatever...

Reply:
to define `displayedColumns` - is approach/requirement of angular cdk table